source/includes/api/_widget.md

```diff
@@ -1,428 +1,296 @@
+> Return all widgets associated with two datasets
 
 ```shell
-curl -X GET https://api.resourcewatch.org/v1/widget/51851e22-1eda-4bf5-bbcc-cde3f9a3a943?includes=user
+curl -X POST "https://api.resourcewatch.org/widget/find-by-ids" \
+-H "Authorization: Bearer <your-token>" \
+-H 'Content-Type: application/json' \
+-d '{
+	"ids": ["<dataset 1 id>", "<dataset 2 id>"]
+}'
 ```
 
 > Example response:
 
 ```json
 {
-    "data": {
-        "id": "51851e22-1eda-4bf5-bbcc-cde3f9a3a943",
-        "type": "widget",
-        "attributes": {
-            "name": "Example Carto Dataset6",
-            "dataset": "be76f130-ed4e-4972-827d-aef8e0dc6b18",
-            "slug": "example-carto-dataset6",
-            "userId": "5820ad9469a0287982f4cd18",
-            "description": null,
-            "source": null,
-            "sourceUrl": null,
-            "authors": null,
-            "application": [
-                "rw"
-            ],
-            "verified": false,
-            "default": false,
-            "protected": false,
-            "defaultEditableWidget": false,
-            "published": true,
-            "freeze": false,
-            "env": "production",
-            "queryUrl": null,
-            "widgetConfig": "{}",
-            "template": false,
-            "layerId": null,
-            "createdAt": "2017-02-08T15:30:34.505Z",
-            "updatedAt": "2017-02-08T15:30:34.505Z",
-            "user": {
-              "name": "John Sample",
-              "email": "john.sample@vizzuality.com"
+    "data": [
+        {
+            "id": "73f00267-fe34-42aa-a611-13b102f38d75",
+            "type": "widget",
+            "attributes": {
+                "name": "Precipitation Change in Puget Sound",
+                "dataset": "06c44f9a-aae7-401e-874c-de13b7764959",
+                "slug": "precipitation-change",
+                "userId": "58333dcfd9f39b189ca44c75",
+                "description": "NOAA nCLIMDIV Precipitation: Historical Precipitation",
+                "source": null,
+                "sourceUrl": null,
+                "authors": null,
+                "application": [
+                    "prep"
+                ],
+                "verified": false,
+                "default": true,
+                "protected": false,
+                "defaultEditableWidget": false,
+                "published": true,
+                "freeze": false,
+                "env": "production",
+                "queryUrl": "query/06c44f9a-aae7-401e-874c-de13b7764959?sql=select annual as x, year as y from index_06c44f9aaae7401e874cde13b7764959%20order%20by%20year%20asc",
+                "widgetConfig": {...}
             }
-        }
-    }
+        },
+        {...}
+    ]
 }
 ```
 
-#### Metadata
-
-Loads the metadata available for the widget. If none are found, an empty array is returned.
+This endpoint allows authenticated users to load all widgets belonging to multiple datasets in a single request. 
```

Comment:
Why do I have to be authenticated for this one, but not the above?

Reply:
This text is a copy-paste from the layers docs. For layers, you do need to be authenticated to use this endpoint. For widgets, you can actually use it without being authenticated. "why" will forever remain a mystery, but I suspect these endpoints are meant to be internal, as they don't paginate responses - AKA you can easily timeout or get a multiple MB json reply using them.

Reply:
I'll update the text so it doesnt say users need to be auth'd